Makefile

```diff
@@ -1048,9 +1049,13 @@ LINT_MD_DOC_FILES = $(shell ls doc/**/*.md)
 run-lint-doc-md = tools/remark-cli/cli.js -q -f $(LINT_MD_DOC_FILES)
 # Lint all changed markdown files under doc/
 tools/.docmdlintstamp: $(LINT_MD_DOC_FILES)
+ifeq ($(NODE_USE_OPENSSL),true)
```

Comment:
Why not simply test `@$(call available-node,"-p require('crypto')")`. Then you can remove all the other "breadcrumb".

Reply:
Or ` -p process.config.variables.node_use_openssl`

Reply:
Good point, it would be safer to do that as the existing `node` binary might have been compiled without openssl (might not be the common case but could happen). I'll take a look and update. Thanks